doc/setup/install.rst

```diff
@@ -378,6 +456,12 @@ Running Odoo
 
 Once all dependencies are set up, Odoo can be launched by running ``odoo.py``.
 
+.. warning:: For the Enterprise edition, you must specify the :file:`enterprise` addons
```

Comment:
Yeah, I think it should be in this doc page; i'll move it from the other one and add a link to here